src/WhatsNewDialog.cpp

```diff
@@ -0,0 +1,214 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  WhatsNewDialog.cpp
+
+  Vitaly Sverchinsky
+
+**********************************************************************/
+
+#include "WhatsNewDialog.h"
+
+#include <wx/fs_mem.h>
+#include <wx/settings.h>
+#include <wx/mstream.h>
+#include <wx/sstream.h>
+#include <wx/txtstrm.h>
+#include <wx/hyperlink.h>
+#include <wx/statline.h>
+#include <wx/checkbox.h>
+#include <wx/frame.h>
+#include <wx/statbmp.h>
+#include <wx/stattext.h>
+
+#include "HelpSystem.h"
+#include "HelpText.h"
+#include "ProjectWindows.h"
+#include "ShuttleGui.h"
+#include "Theme.h"
+
+#include "../images/WhatsNewBtn.jpeg.h"
+
+namespace
+{
+
+const char* WhatsNewURL = "https://audacityteam.org/3.5.0-video";
+const char* PluginsURL = "";
+const char* MuseHubURL = "";
+
```

Comment:
MusehubURL and PluginsURL are identical. rename instances of PluginsURL to MuseHubURL and set MuseHubURL "https://www.musehub.com"